src/helper_input.R

```diff
@@ -36,9 +36,21 @@ collapseStory <- function() {
     ), # bsCollapsePanel
     bsCollapsePanel(
       title = "2. Pre-existing Conditions",
-      checkboxInput('is_sick', div("I have ", tags$a("flu-like symptoms", href = urls$cdc_symptoms))),
+      checkboxInput('is_sick', div("I have ", tags$a("symptoms of COVID", href = urls$cdc_symptoms))),
```

Comment:
@pendlrobinson could you change "symptoms of COVID" to "potential symptoms of COVID-19"?